### PR TITLE
Add acceptance test for Dataproc Image 2.2 and spark runtime version 2.2

### PR DESCRIPTION
**Pull request by isha97**

### CODE REVIEW

**Review by davidrabinowitz — APPROVED** [review]

File: spark-bigquery-dsv1/spark-bigquery-with-dependencies_2.11/src/test/java/com/google/cloud/spark/bigquery/acceptance/Scala211DataprocImage13AcceptanceTest.java [hunk 1]
```diff
@@ -18,7 +18,9 @@
 import java.util.Collections;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.junit.Ignore;
 
+@Ignore
 public class Scala211DataprocImage13AcceptanceTest extends DataprocAcceptanceTestBase {
```
Comment (vishalkarve15) [inline spark-bigquery-dsv1/spark-bigquery-with-dependencies_2.11/src/test/java/com/google/cloud/spark/bigquery/acceptance/Scala211DataprocImage13AcceptanceTest.java:24]: Let's delete this file and other such files instead.

File: spark-bigquery-dsv2/spark-2.4-bigquery/src/test/java/com/google/cloud/spark/bigquery/acceptance/Spark24DataprocImage14AcceptanceTest.java [hunk 2]
```diff
@@ -18,7 +18,9 @@
 import java.util.Collections;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.junit.Ignore;
 
+@Ignore
```
Comment (vishalkarve15) [inline spark-bigquery-dsv2/spark-2.4-bigquery/src/test/java/com/google/cloud/spark/bigquery/acceptance/Spark24DataprocImage14AcceptanceTest.java:23]: Let's delete these files.

File: spark-bigquery-dsv1/spark-bigquery-with-dependencies_2.11/src/test/java/com/google/cloud/spark/bigquery/acceptance/Scala211DataprocImage13AcceptanceTest.java (hunk 1 above)
Comment (isha97) [reply spark-bigquery-dsv1/spark-bigquery-with-dependencies_2.11/src/test/java/com/google/cloud/spark/bigquery/acceptance/Scala211DataprocImage13AcceptanceTest.java:24]: David suggested to keep these files for now.

File: spark-bigquery-dsv1/spark-bigquery-with-dependencies_2.12/src/test/java/com/google/cloud/spark/bigquery/acceptance/Scala212DataprocImage22AcceptanceTest.java [hunk 3]
```diff
@@ -24,7 +24,8 @@ public class Scala212DataprocImage22AcceptanceTest extends DataprocAcceptanceTes
   private static AcceptanceTestContext context;
 
   public Scala212DataprocImage22AcceptanceTest() {
-    super(context);
+    // TODO: sparkStreamingSupported should be set to true onceå bug is fixed in image 2.2
```
Comment (davidrabinowitz) [inline spark-bigquery-dsv1/spark-bigquery-with-dependencies_2.12/src/test/java/com/google/cloud/spark/bigquery/acceptance/Scala212DataprocImage22AcceptanceTest.java:27]: nit: /onceå/once/

File: spark-bigquery-connector-common/src/test/java/com/google/cloud/spark/bigquery/acceptance/DataprocServerlessAcceptanceTestBase.java [hunk 4]
```diff
@@ -65,11 +69,24 @@ public void createBatchControllerClient() throws Exception {
...
+        BatchControllerSettings.newBuilder().setEndpoint(AcceptanceTestConstants.DATAPROC_ENDPOINT);
+    TimedRetryAlgorithm timedRetryAlgorithm =
+        OperationTimedPollAlgorithm.create(
+            RetrySettings.newBuilder()
+                .setInitialRetryDelay(Duration.ofMillis(5000L))
+                .setRetryDelayMultiplier(1)
+                .setMaxRetryDelay(Duration.ofMillis(45000L))
+                .setInitialRpcTimeout(Duration.ZERO)
+                .setRpcTimeoutMultiplier(1.0)
+                .setMaxRpcTimeout(Duration.ZERO)
```
Comment (davidrabinowitz) [inline spark-bigquery-connector-common/src/test/java/com/google/cloud/spark/bigquery/acceptance/DataprocServerlessAcceptanceTestBase.java:82]: Please set to 600s as well

**Review by davidrabinowitz — APPROVED** [review]

**Pull request merged**